storage/tests/unit/test_bucket.py

```diff
@@ -1166,27 +1166,12 @@ def test_copy_blobs_preserve_acl(self):
         dest = self._make_one(client=client, name=DEST)
         blob = self._make_blob(SOURCE, BLOB_NAME)
 
-        new_blob = source.copy_blob(
-            blob, dest, NEW_NAME, client=client, preserve_acl=False
-        )
-
-        self.assertIs(new_blob.bucket, dest)
-        self.assertEqual(new_blob.name, NEW_NAME)
-        self.assertIsInstance(new_blob.acl, ObjectACL)
-
-        kw1, kw2 = connection._requested
-        COPY_PATH = "/b/{}/o/{}/copyTo/b/{}/o/{}".format(
-            SOURCE, BLOB_NAME, DEST, NEW_NAME
+        source.copy_blob(blob, dest, NEW_NAME, client=client, preserve_acl=False)
+        mock_warn.assert_called_once_with(
+            "The 'preserve_acl' argument is deprecated. Please update the bucket's ACL manually.",
```

Comment:
Please don't assert the warning message (it also happens to be incorrect, as the ACL is for the blob, not its bucket).